StyleCop.Analyzers/StyleCop.Analyzers.Test/OrderingRules/SA1201UnitTests.cs

```diff
@@ -143,6 +144,28 @@ public class TestClass { }
             };
 
             await this.VerifyCSharpDiagnosticAsync(testCode, expected, CancellationToken.None).ConfigureAwait(false);
+
+            string fixedCode = @"public class OuterType
+{
+    public string TestField;
+    public OuterType() { }
+    ~OuterType() { }
+    public delegate void TestDelegate();
+    public event TestDelegate TestEvent { add { } remove { } }
+    public enum TestEnum { }
+    public interface ITest { }
+    public string TestProperty { get; set; }
+    public string this[string arg] { get { return ""foo""; } set { } }
+    public static explicit operator bool(OuterType t1) { return t1.TestField != null; }
+    public static OuterType operator +(OuterType t1, OuterType t2) { return t1; }
+    public void TestMethod () { }
+    public struct TestStruct { }
+    public class TestClass { }
+}
```

Comment:
This test case could be improved by not having TestClass at the correct location in the testCode.